schema-sandbox/schema.json

```
@@ -30,15 +30,22 @@
             "minItems": 1,
             "items": {
                 "anyOf": [
-                    {"$ref": "#definitions/person"},
-                    {"$ref": "#definitions/entity"}
+                    {"$ref": "#/definitions/person"},
+                    {"$ref": "#/definitions/entity"}
                 ]
             }
         },
         "commit": {
             "type": "string",
             "description": "The commit SHA1 hash or revision number of the software version",
             "pattern": "^[a-f0-9]{7,40}$"
+        },
+        "identifiers": {
+            "items": {
+                "$ref": "#/definitions/identifier"
+            },
+            "minItems": 1,
```

Shouldn't this also be unique items?

```suggestion
            "minItems": 1,
             "uniqueItems": true,
```

---

added